### DISCUSSION TITLE
Team VS Score Balancing Option

### DISCUSSION THREAD

**Discussion by ToamtoSauce** [ppy/osu#31149] [Ideas]
would be a option to toggle when creating a mutiplayer room (i quickly made a very janky mockup)

![image](https://github.com/user-attachments/assets/929e2e27-685a-4b60-830f-709ef1dac04e)

when choosing Team VS, have an option to balance team scores
(e.g. if there's 1 player on the red team, and 3 players on the blue team, the red team's score would be multiplied by 3 to account for the lack of teammates on their team)

(other example)
red team - 2 players
blue team - 7 players
red team's score multiplied by 3.5

this would only affect team score and not scores submitted to leaderboards obv

**Comment by peppy** [comment]
Definitely want to see this one.
